frontends/web/src/components/icon/icon.tsx

```diff
@@ -112,6 +113,7 @@ export const ArrowCirlceLeftActive = (props: ImgProps) => (<img src={arrowCircle
 export const ArrowCirlceRight = (props: ImgProps) => (<img src={arrowCircleRightSVG} draggable={false} {...props} />);
 export const ArrowCirlceRightActive = (props: ImgProps) => (<img src={arrowCircleRightActiveSVG} draggable={false} {...props} />);
 export const Bank = (props: ImgProps) => (<img src={bankSVG} draggable={false} {...props} />);
+export const BuyInfo = (props: ImgProps) => (<img src={buyInfoSVG} draggable={false} {...props} />);
```

Comment:
The reason I didn't use  `Info` was because:
1. `Info` is already used by the other Info icon (the one you encircled on the menu).
2. And because they're not actually the same shape.

![Screenshot 2023-01-10 at 02 37 30](https://user-images.githubusercontent.com/28676406/211442642-15e443ec-e415-488a-a968-2988dd62e2d2.png)

3. Thus only used in the context of buy -> `BuyInfo`

We can rename it to smt like InfoDark or similar if you want?

Reply:
thanks for zooming in, imo it doesn't make any sense to have 2 different ones, especially if the difference is hardly visible, but up to you and @jadzeidan